ros2interface/test/test_cli.py

```diff
@@ -15,7 +15,6 @@
 import contextlib
 import itertools
 import re
-import sys
```

Comment:
You still need `sys` for elsewhere in the file.

Reply:
Oops, done in https://github.com/ros2/ros2cli/pull/583/commits/4018dec5fefa467a878414b74083ed890cbd8811.